dandi/tests/test_validate.py

```diff
@@ -16,6 +17,14 @@ def test_validate_nwb_error(simple3_nwb: Path) -> None:
     assert len([i for i in validation_result if i.severity]) > 0
 
 
+def test_validate_relative_path(bids_examples: Path) -> None:
+    selected_dataset_path = bids_examples / "asl003"
+    os.chdir(selected_dataset_path)
```

Comment:
Use pytest's `monkeypatch` fixture instead of changing the PWD for all tests.

Reply:
ok.